manager/controllers/cves/details.go

```diff
@@ -0,0 +1,81 @@
+package cves
+
+import (
+	"app/base/logging"
+	"app/base/models"
+	"app/base/utils"
+	"app/manager/base"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type GetCveDetailsSelect struct {
+	Cvss2Score    *float32         `json:"cvss2_score"`
+	Cvss2SMetrics *string          `json:"cvss2_metrics"`
+	Cvss3Score    *float32         `json:"cvss3_score"`
+	Cvss3Metrics  *string          `json:"cvss3_metrics"`
+	Description   *string          `json:"description"`
+	Severity      *models.Severity `json:"severity"`
+	PublicDate    *time.Time       `json:"publish_date"`
+	Name          *string          `json:"synopsis"`
+	RedHatURL     *string          `json:"redhat_url"`
```

Comment:
```RedHatURL``` gorm maps as ```red_hat_url```, and it returns empty string value, it should be ```RedhatURL``` in go code or you can add gorm tag \`gorm:"column:redhat_url"\`

Reply:
oh, thanks, nice catch :D

Reply:
@yungbender Why is everything a pointer, even values that can't be null/nil? I see it is the same in the already merged endopoint. I think only values that can be nil should be allowed to be pointer it can prevent some errors.

Reply:
@jdobes sorry, did not notice this comment, sure we can do it like that. But what error from pointer side can happen when value cannot be null in the db?

Reply:
I wanted the models, to be simplest as possible and not mix this value/pointer approach together. Also, with this mixed approach you cannot forget to change the pointer to the value (or vice-versa) when you change the db conditions.

Reply:
@yungbender you might be correct, I think it's better to have structs defined as strictly as possible, but it also means more code to maintain, let's keep it as is now if it doesn't cause any issues